app/form_models/jobseekers/profile/qualified_teaching_status_form.rb

```diff
@@ -0,0 +1,10 @@
+class Jobseekers::Profile::QualifiedTeachingStatusForm
+  include ActiveModel::Model
```

Comment:
Don't need to include this module if you inherit from BaseForm I think

Reply:
changed about you form too